docs/features.md

```diff
@@ -119,12 +120,17 @@ Format your code with various Haskell code formatters.
 | Ormolu          | `hls-ormolu-plugin`          |
 | Stylish Haskell | `hls-stylish-haskell-plugin` |
 
+---
+
 Format your cabal files with a cabal code formatter.
+The default cabal code formatter is `cabal-gild`, which needs to be available on the `$PATH`,
```

Comment:
This isn't a great experience, actually. So the default behaviour will send people errors, since most people won't have installed this? I think our default behaviour shouldn't rely on any external programs, which makes me think we should set the default cabal formatter to "none".

Reply:
Hm, I thought we did. We definitely should have a "none" option.

Reply:
Yeah, the initial experience is not good. In the case of cabal-gild, we can likely ship it with all supported GHC versions.

However, perhaps we should start shipping the binaries instead of linking it into HLS directly. Then every HLS version would have the same formatter version. Let's give #411 another kick.